src/linux/Packaging.Linux/install-from-source.sh

```diff
@@ -0,0 +1,176 @@
+#!/bin/sh
+
+# halt execution immediately on failure
+# note there are some scenarios in which this will not exit;
+# see https://www.gnu.org/software/bash/manual/html_node/The-Set-Builtin.html
+# for additional details
+set -e
+
+is_ci=false
+for i in "$@"; do
+    case "$i" in
+        -y)
+        is_ci=true
+        shift # past argument=value
+        ;;
+    esac
+done
+
+# in non-ci scenarios, advertise what we will be doing and
+# give user the option to exit
+if [ ! "$is_ci" ]; then
+    echo "This script will download, compile, and install Git Credential Manager to:
+
+    ~/usr/local/bin
```

Comment:
I was initially confused by this `~`, but you _do_ add `~/usr/local/bin` into the `PATH`, so :+1:.